.github/workflows/backend_override.tf

```diff
@@ -0,0 +1,5 @@
+terraform {
```

Comment:
Can you create a directory called `ci` and move this into that? I think leaving only github workflows in the workflow directory is cleaner.

Reply:
Done